static/js/components/SourceTable.jsx

```diff
@@ -321,6 +322,17 @@ const SourceTable = ({
               )}
               {!source.photometry_exists && <div> no photometry exists </div>}
             </Grid>
+            <Grid item>
+              {source.color_magnitude.length && (
+                <Suspense fallback={<div>Loading HR diagram...</div>}>
+                  <VegaHR
+                    data={source.color_magnitude}
+                    width={200}
+                    height={200}
```

Comment:
Should the width/height be set by the context (ie. a percentage of the screen, rather than size; mobile vs desktop)?

Reply:
I'm not sure. I think the component should not be deciding for itself, rather let the surrounding page choose the size for it.